R/euclidean_logical_joins.R

```diff
@@ -1,203 +1,68 @@
-#' Spatial Anti Join Using LSH
+#' Spatial joins Using LSH
 #'
-#' @param a the first dataframe you wish to join.
-#' @param b the second dataframe you wish to join.
+#' @inheritParams jaccard_left_join
+#' @param threshold The distance threshold below which units should be
+#'   considered a match. Note that contrary to Jaccard joins, this value is
+#'   about the distance and not the similarity. Therefore, a lower value means a
+#'   higher similarity.
+#' @param r Hyperparameter used to govern the sensitivity of the locality
+#'   sensitive hash.
```

Comment:
```suggestion
#' @param r Hyperparameter used to govern the sensitivity of the locality
#' sensitive hash. Corresponds to the width of the hash bucket in the LSH
#' algorithm. Increasing values of r mean more hash collisions and higher
#' sensitivity (fewer false-negatives) at the cost of lower specificity (more
#' false-positives and longer run time). For more information, see
#' <doi:10.1145/997817.997857>.
```

This all looks great to me! Tying to add a suggestion for some expanded documentation about the `r` parameter in GitHub Codespaces, but it's my first time using this feature so sorry if the formatting is off.

Reply:
Yes it worked great. I ended up copying your suggested change and making the commit myself to add some markdown formatting and run `devtools::document()` but otherwise it was fine

Reply:
superb! merging in now